### PR TITLE
I232: Strange UI response on Zoom-out

### LINKED ISSUE #232: Strange UI responsiveness on zoom-out
When zooming out (ctrl-) in the Album Grid view, the whole GUI zooms except the images (at least not properly). At least when going back from 100% to 25%. The images should be responsive too. Can we use % and scaling instead of static image sizes? 

Also with my 1920x1080 resolution in Windows with a default 125% Zoom in Windows, the GUI is just too high and falls behind the play bar. I think we should make it a bit more responsive to work better in different resolutions and settings.

### PR DESCRIPTION
**Pull request by Dnouv**
As mentioned in issue #232, while zooming out, the cover image was not responsive; the issue is fixed in this PR; the cover image is now responsive; that is, it becomes small in size while zooming out. Please refer to the screenshots below for the new changes. (Fixes #232 )
The error page
navid232_err

Fixed page:
navid232_fix

Feedbacks are most welcomed. Thank you!

### CODE REVIEW

**Comment by deluan** [comment]
Thanks for trying to fix this. But with zoom in 100% (normal view), the cover sizes are completely wrong:
### Master
Screen Shot 2021-03-25 at 11 01 10 PM

### Your changes
Screen Shot 2021-03-25 at 11 00 57 PM

**Comment by Dnouv** [comment]
@deluan, I have tried to fix the issue with the normal view; please have a look.
On demo page(normal view):
navid232nberr

On fixed page(normal view):  
navid232nbf

Feedbacks are most welcomed; thank you!

File: ui/src/album/AlbumGridView.js [hunk 1]
```diff
@@ -85,7 +103,7 @@ const useCoverStyles = makeStyles({
 const getColsForWidth = (width) => {
   if (width === 'xs') return 2
   if (width === 'sm') return 3
-  if (width === 'md') return 4
+  if (width === 'md') return 5
```
Comment (deluan) [inline ui/src/album/AlbumGridView.js:106]: Any particular reason to change this?

**Review by deluan — COMMENTED** [review]
I'd like to have another opinion about this...

Hey @jvoisin (or anyone else that wants to give an opinion: Can you download this version from https://github.com/navidrome/navidrome/suites/2351867355/artifacts/49729439 and try it out?

File: ui/src/album/AlbumGridView.js (hunk 1 above)
Comment (Dnouv) [reply ui/src/album/AlbumGridView.js:106]: There is no particular reason; actually, while debugging, I had made these changes and forgot to change them back; I will restore it to the previous value; sorry for the trouble, and thanks for your review.

**Comment by Dnouv** [comment]
@deluan, I understand you may be busy, but it would be nice if you could review this PR.
Thank you!

**Comment by deluan** [comment]
I don't think the solution for the whole "responsive" album grid is just make the images scale down. IMHO, the end result is even worse:
Original:
Screen Shot 2021-04-20 at 9 38 54 PM

Your changes:
Screen Shot 2021-04-20 at 9 39 03 PM

I think the whole album grid must be responsive, like dynamically changing the number of alums per row. I rather keep the current behaviour (it is the same behaviour as Plex), and take this into consideration when reworking the Album Grid (#608)

**Comment by Dnouv** [comment]
Thanks for your review, @deluan. We could try scaling up the images and increasing the number of albums per row for this screen size. I think this will fulfill both the criteria similar to plex and more responsive.

**Comment by deluan** [comment]
I think we should close this for now, and get back to it once we start the work on the new Album Grid. The code is already messy enough and this may make it even harder to comprehend.

**Comment by Dnouv** [comment]
Oh, you mean the GSoC Album Grid project. Ok, let us wait for the start of the project.

**Comment by deluan** [comment]
Hey, I'm closing this as I'll need to rewrite the UI with the new React-Admin version (v4) and will try to implement this in a different way. Thanks anyway for the contribution!

**Pull request closed without merging**